bower.json

```diff
@@ -25,6 +25,7 @@
     "purescript-lists": "^0.7.0",
     "purescript-random": "^0.2.0",
     "purescript-strings": "^0.7.0",
-    "purescript-transformers": "^0.8.1"
+    "purescript-transformers": "^0.8.1",
+    "purescript-lens": "~0.9.0"
```

Comment:
Lens isn't in the core library set, and relatively infrequently updated AFAIK, so I'd prefer not to add it as a dependency here.